Library/Formula/racket.rb

```diff
@@ -28,17 +31,43 @@ def install
       system "make"
       system "make", "install"
     end
+
+    # configure racket's package tool (raco) to do the Right Thing
+    # see: http://docs.racket-lang.org/raco/config-file.html
+    inreplace etc/"racket/config.rktd" do |s|
+        s.gsub!(
+            /\(bin-dir\s+\.\s+"#{Regexp.quote(bin)}"\)/,
+            "(bin-dir . \"#{HOMEBREW_PREFIX}/bin\")",
+        )
+        s.gsub!(
+            /\n\)$/,
+            "\n      (default-scope . \"installation\")\n)",
+        )
+    end
   end
 
   def caveats; <<-EOS.undent
     This is a minimal Racket distribution.
-    If you want to use the DrRacket IDE, we recommend that you use
-    the PLT-provided packages from http://racket-lang.org/download/.
+    If you want to use the DrRacket IDE, you may run
+      raco pkg install --auto drracket
     EOS
   end
 
   test do
     output = shell_output("#{bin}/racket -e '(displayln \"Hello Homebrew\")'")
     assert_match /Hello Homebrew/, output
+
+    # show that the config file isn't malformed
+    output = `'#{bin}/raco' pkg config`
+    assert $?.success?
+    assert_match Regexp.new("^name:
```

Comment:
brew test locally doesn't reflect any changes made on this line unless I uninstall, reinstall.
It's the install that takes ten minutes.

I don't see " just a / instead of Regexp.new". Did you change your mind on that bit?